app/controllers/trains_controller.rb

```diff
@@ -25,6 +25,6 @@ def update
 
   def destroy
     response = TrainApi.new.destroy(params[:id])
-    redirect_to trains_path
+    redirect_to trains_path, notice: response["message"]
```

Comment:
oh i see we never did this for the other controllers.
well then, this is still a good one to add haha